app/code/Magento/Contact/Controller/Index/Post.php

```diff
@@ -101,4 +65,61 @@ private function getDataPersistor()
 
         return $this->dataPersistor;
     }
+
+    /**
+     * @param array $post Post data from contact form
+     * @return void
+     */
+    protected function sendEmail($post)
```

Comment:
Magento discourages the introduction of any new "protected" properties or methods. The reason is that we are trying to get rid of inheritance-based APIS and promote composition ver inheritance. Only public or private modifiers should be used.

Please change this to "private"